src/server.ts

```diff
@@ -14,6 +15,9 @@ import pageRouter from "./routes/pages";
 const forceHttp: boolean =
     process.env.LINEUPLIST_FORCE_HTTP && process.env.LINEUPLIST_FORCE_HTTP === "true" ? true : false;
 
+// Windows often has port 80 already in use for various system services, so default to 8080
```

Comment:
You an use 8080 as the macOS http port too, no need to differentiate.

Reply:
OK, just changed the hardcoding to 8080